source/conditionals.rst

```diff
@@ -17,7 +20,7 @@ Furthermore conditional loops enable us to harness logic relating to repetition.
 Code Blocks
 ===========
 
-A block of code is code that will execute together. A block is defined by the 
+A block of code is code that will execute together. In python a block is defined by the
```

Comment:
Minor, but I've been trying to keep it **P**ython for language to differentiate from **p**ython the interpreter.